catboost/pytest/test.py

```diff
@@ -5642,3 +5642,55 @@ def test_broken_dsv_format(dataset_name, loss_function, has_pairs, has_group_wei
 
     with pytest.raises(yatest.common.ExecutionError):
         yatest.common.execute(cmd)
+
+
+@pytest.mark.parametrize('pool', ['higgs'])
```

Comment:
If you pass only one value for `pool`, `pytest.mark.parametrize` is not needed then, I suppose :)

Reply:
True. It was supposed to work for another pool too, but that's still in progress. Thanks for pointing out.